rxdogtag/build.gradle

```diff
@@ -26,4 +27,13 @@ dependencies {
   testCompile deps.test.truth
 }
 
+jmh {
+  jmhVersion = deps.versions.jmhLib
+
+  if (project.hasProperty("jmh")) {
```

Comment:
For the sake of simplicity since it's small right now, let's remove this

Reply:
I'm fine removing this, though I found it to be a very useful trick when experimenting with different setups and seeing how it affected runs. :)

Reply:
Hmmm, fair point. Let's keep it then but maybe add a comment above it with instructions on how to run